master/usage/removing-nodes.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Removing Nodes
+---
+
+With each Calico node there is a Node entry in the datastore and Workload
+Endpoints are associated with that node.  As nodes are replaced (with
+differently named nodes) or decomissioned, the datastore will accumulate
+node entries and possibly workload entries.  Currently there is not an
+automatic reaping of these old Node and Endpoint entries so it is necessary
+to periodically clean up these old entries or ensure they are removed when
+nodes are decomissioned.  This page provides instruction of how to remove
+those entries.
+
+> **Note**: Removing a Node entry will also remove the Workload Endpoint
```

Comment:
We probably need to state somewhere that the calico/node container should no longer be running on that host since it automatically adds in a Node resource for that Node.  Therefore the first stage of decommissioning should be shut down the calico/node container and to ensure it is not restarted.  Restarting the container will cause a corresponding Node resource to be re-added to the Calico configuration.